src/intersection-observer-polyfill.js

```diff
@@ -38,6 +38,8 @@ export const DEFAULT_THRESHOLD =
     [0, 0.05, 0.1, 0.15, 0.2, 0.25, 0.3, 0.35, 0.4,
     0.45, 0.5, 0.55, 0.6, 0.65, 0.7, 0.75, 0.8, 0.85, 0.9, 0.95, 1];
 
+const INIT_TIME = Date.now();
```

Comment:
Oh, it's because I'm [only checking](https://github.com/ampproject/amphtml/blob/master/build-system/eslint-rules/no-export-side-effect.js#L19) exports. While this won't be DCEd, it's fine as is.